todo_react/todo-react/src/App.tsx

```diff
@@ -0,0 +1,55 @@
+import React, {useState} from 'react';
+import './App.css';
+import {AddTask} from "./components/AddTask";
+import {ToDoItem} from "./components/ToDoItem";
+import {ITodo} from "./models";
+
+export const App: React.FC = () => {
+    let todoList = [...JSON.parse(localStorage.getItem('TodoList') || "")]
+    const [todos, setTodos] = useState<Array<{ id: number, completed: boolean, title: string }>>(todoList)
```

Comment:
try to move `todos` and function that changes it to [React Context](https://reactjs.org/docs/context.html) for practice